Automattic/jetpack
# js-tools: Replace deprecated tweetsodium dep

#### Changes proposed in this Pull Request:

It recommends libsodium-wrappers, so let's go with that.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Try the script (on a test repo, please), see that it still works and updates the secret correctly.
  * :heavy_check_mark: https://github.com/anomiex/testing/runs/5978093879?check_suite_focus=true#step:4:6
